modules/clas/src/main/ui/StudentFormUi.scala

```diff
@@ -217,6 +217,13 @@ final class StudentFormUi(helpers: Helpers, clasUi: ClasUi, studentUi: StudentUi
                 cls   := "button button-empty button-red",
                 title := trans.clas.closeDesc1.txt()
               )(trans.clas.closeStudent())
+            ),
+            (!s.student.managed).option(
```

Comment:
why this check?

Reply:
I coded the move for the managed student first and then I realized it was making sense for the non managed student too.

The views of managed and non managed students are a bit different. 
When you click on a managed student in the list, you see 2 sets of actions 

![image](https://github.com/user-attachments/assets/0f6fb2cf-694d-424b-847f-f476b9218a9c)
![image](https://github.com/user-attachments/assets/d6946fa4-b8b1-447e-af1c-40b516d92dc2)

When you click on a non managed student you only have the first one
So I had a choice:
- Overloading the first set of actions with the move (keeping consistency between the 2 types of students)
- Do it one click later  for the non managed in the edit form (but hiding it for the managed student because it was useless then)

So if you have a strong opinion about where it should be for this PR, I am happy to adapt.